build.gradle

```diff
@@ -61,5 +61,6 @@ repositories {
 
 dependencies {
     compile('org.yaml:snakeyaml:1.23')
+    compile('com.jayway.jsonpath:json-path:2.4.0')
```

Comment:
this is interesting.  i'm not sure how we should manage our dependencies.  does everyone need to include all libs? is there a core json parser?  we can worry about this later.